data/genomics/homo_sapiens/illumina/10xgenomics/cellranger_multi/README.md

```diff
@@ -0,0 +1,70 @@
+# cellranger multi test data
+
+This folder contains test data for the `cellranger multi` module.
+Tests for `cellranger multi` will also refer to FASTQs from B cells in the [sister folder](../cellranger_vdj/README.md) for the `cellranger vdj` module.
+This folder contains the corresponding 5' gene expression FASTQs for the B cell data.
```

Comment:
would it make sense to symlink them? It might be clearer to have a complete dataset in the cellranger multi folder ... but I'm also not sure if this is something that is done in this repo in general.

Reply:
symlinks added, README updated accordingly

does this mean that the nf-core/modules tests can traverse the symlinks? that would facilitate testing a `cellranger multi` module

Reply:
I would hope so! But I don't know for sure...

Reply:
understood. I'll see what happens with the module tests and report back 👍🏼